src/app/event/event-detail-controller.js

```diff
@@ -1,9 +1,8 @@
 angular.module('songster.event')
 
-    .controller('EventDetailController', function EventCtrl($scope,$http,$account, $location, $event, $state, $rootScope) {
+    .controller('EventDetailController', function EventCtrl($scope,$http, $location, $event, $state, $rootScope) {
         $scope.event = $event.getEvent();
-        $scope.user = $account.getUser();
-        $scope.isDJ =  $account.getUser() !== null && $event.getEvent().owner_id ==  $account.getUser()._id;//$rootScope.isDj();
+        $scope.isDJ =  $rootScope.isDj();
```

Comment:
it is not necessary to assign it to a scope variable, because the isDj() method is already available via the rootScope in the html